org.kicad.KiCad.Library.Templates.yml

```diff
@@ -0,0 +1,40 @@
+id: org.kicad.KiCad.Library.Templates
+branch: '5.1'
+runtime: org.kicad.KiCad
+runtime-version: stable
+sdk: org.freedesktop.Sdk//20.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+modules:
+- name: kicad-templates
+  buildsystem: simple
+  build-commands:
+  - install -D apply_extra $FLATPAK_DEST/bin/apply_extra
+  post-install:
+  - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo org.kicad.KiCad.Library.Templates.metainfo.xml
+  - appstream-compose --basename=org.kicad.KiCad.Library.Templates --prefix=${FLATPAK_DEST}
+    --origin=flatpak org.kicad.KiCad.Library.Templates
+  sources:
+  - type: script
+    commands:
+    - tar xzf kicad-templates.tar.gz --strip-components 1 --wildcards "*/Projects/*" "*.kicad_wks"
+    - mkdir template
+    - find ./Worksheets/ -type f -name '*.kicad_wks' -exec mv {} template/ \;
+    - mv Projects/* template/
+    - mv kicad.pro template/
+    - rm -rf kicad-templates.tar.gz Projects/ Worksheets/
+    dest-filename: apply_extra
+  - type: extra-data
+    filename: kicad-templates.tar.gz
+    url: https://gitlab.com/kicad/libraries/kicad-templates/-/archive/5.1.9/kicad-templates-5.1.9.tar.gz
+    sha256: 0c1bf3d2e6d8d1056a5da6c1f7a173551c154b4bdaddb86b6a34155b18e65da6
+    size: 1643954
+    installed-size: 5877944
+  - type: extra-data
```

Comment:
Why does it use extra-data instead of embedding it into the extension?

Reply:
I used extra-data, because for the huge existing .Packages3D extension (~900MB), the aarch64 builds timed out in about 9 out of 10 tries. When I switched this to extra-data, it worked 100% of the time.

Coming from this, I thought I had to use extra-data as well for the new extensions in order to make merge-dirs and the directory layout work.

I will happily switch all of them to store the data inside the extensions themselves. This would be so much easier, really. But I fear that this would break aarch64 builds again.

Do you know how to avoid timing out builds with larger builds?